shell/components/PromptRemove.vue

```diff
@@ -105,8 +105,8 @@ export default {
         return null;
       }
 
-      if (this.toRemove[0].doneLocationRemove) {
-        return this.toRemove[0].doneLocationRemove;
```

Comment:
`doneLocationRemove` is no longer used in our codebase.

It was originally added to support Epinio https://github.com/rancher/dashboard/commit/917ce2ef0199b25e8aa47a1aee087017fa552b53

We do need to respect `doneOverride` here just like we do in other locations so I decided to swap the two.